apps/namadillo/src/hooks/useTransactionCallbacks.tsx

```diff
@@ -19,4 +21,27 @@ export const useTransactionCallback = (): void => {
   useTransactionEventListener("Bond.Success", onBalanceUpdate);
   useTransactionEventListener("Unbond.Success", onBalanceUpdate);
   useTransactionEventListener("Withdraw.Success", onBalanceUpdate);
+
+  const store = createStore();
```

Comment:
It might be better to use `getDefaultStore` here in case you get the same problems as in #1007.